src/schedule_at.ts

```diff
@@ -0,0 +1,45 @@
+import { AlgebraicType, SumTypeVariant } from "./algebraic_type";
+import { AlgebraicValue } from "./algebraic_value";
+
```

Comment:
AFAIK this file should be in client_api folder

Reply:
No, it's not part of the WS schema. the `client_api` folder is specifically for definitions generated from `client-api-messages` in the SpacetimeDB repo.